packages/react/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Form } from ".";
+import { ConfigurationProvider } from "../../context/config-context";
+
+const meta: Meta<typeof Form> = {
+  component: Form,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Form>;
+
+export const Primary: Story = {
+  render: () => (
+    <ConfigurationProvider>
+      <Form
+        middleware={({ user }) => {
+          console.log("middleware - user: ", user);
+          return new Promise((resolve) => {
+            resolve(user);
+          });
+        }}
+        factors={[{ method: "email_link" }, { method: "otp_via_sms" }]}
+        text={{
+          "initial.title": "Multi-Factor Authentication",
```

Comment:
Similar to other feedback, as the default story this feels out of place.

I would suggest that for the default story we show the absolute minimum implementation which is:
```ts
<Form />
```

What do you think?

Reply:
yes you're right, I will change it 👍